runtime/doc/options.txt

```diff
@@ -4872,6 +4872,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 <	Replace the ';' with a ':' or whatever separator is used.  Note that
 	this doesn't work when $INCL contains a comma or white space.
 
+							   *'plugindir'*
+'plugindir' 'pldir'	string (default "plugged")
```

Comment:
Short option should be `'pidir` ("pi" is the unofficial Vim abbreviation for "plugin"[1]). Default should probably be "bundle", since that is overwhelmingly the common case.

---

[1] illustrated by the existence of these helpdocs:

    pi_tar.txt  pi_zip.txt  pi_spec.txt  pi_gzip.txt  pi_paren.txt  
    pi_netrw.txt  pi_vimball.txt  pi_matchit.txt  pi_getscript.txt

And also the vim-pi project.